<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettings</class>
 <widget class="QWidget" name="AppearanceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="generalBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="themaLabel">
            <property name="text">
             <string>Theme</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="themeComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="windowStateCheckBox">
          <property name="text">
           <string>Save window size and position on exit.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="fontSizeBox">
       <property name="title">
        <string>Font Size</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="fontSizeTextEditorLabel">
            <property name="text">
             <string>Text Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fontSizeTextEditorLabelSpinBox">
            <property name="minimum">
             <number>9</number>
            </property>
            <property name="maximum">
             <number>18</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="fontSizeInformationBoardLabel">
            <property name="text">
             <string>Information Board</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fontSizeInformationBoardSpinBox">
            <property name="minimum">
             <number>9</number>
            </property>
            <property name="maximum">
             <number>18</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="toolbarIconBox">
       <property name="title">
        <string>Toolbar Icon</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="toolbarIconSizeLabel">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="smallRadioButton">
            <property name="text">
             <string>small</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mediumRadioButton">
            <property name="text">
             <string>medium</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="largeRadioButton">
            <property name="text">
             <string>large</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="toolbarIconStyleLabel">
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="justTextRadioButton">
            <property name="text">
             <string>just text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="justIconRadioButton">
            <property name="text">
             <string>just icons</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="textAndIconsRadioButton">
            <property name="text">
             <string>text and icons</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Crypto Operations</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0" rowminimumheight="0,0,0,0,0,0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <item row="0" column="1">
             <widget class="QCheckBox" name="decrCheckBox">
              <property name="text">
               <string>Decrypt</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QCheckBox" name="decrVerifyCheckBox">
              <property name="text">
               <string>Decrypt Verify</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="signCheckBox">
              <property name="text">
               <string>Sign</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="verifyCheckBox">
              <property name="text">
               <string>Verify</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="encrCheckBox">
              <property name="text">
               <string>Encrypt</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QCheckBox" name="encrSignCheckBox">
              <property name="text">
               <string>Encrypt Sign</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QCheckBox" name="symmetricEncrCheckBox">
              <property name="text">
               <string>Sym. Encrypt</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
